lib/readers/Transformer.js

```diff
@@ -1,21 +1,22 @@
-const AbstractReader = require("../AbstractReader");
+import AbstractReader from "../AbstractReader.js";
 
 /**
  * A reader that allows modification of all resources passed through it.
  *
- * @public
- * @memberof module:@ui5/fs.readers
- * @augments module:@ui5/fs.AbstractReader
+ * @private
```

Comment:
We're setting the transformer here to private because we plan to remove this artifact completely in one of the next steps.

Reply:
Only a note for anyone who is wondering about 🧙🏽‍♂️